workbench-core/environments/src/interfaces/environmentTypeConfig.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { z } from 'zod';
+
+// eslint-disable-next-line @rushstack/typedef-var
+export const EnvironmentTypeConfigParser = z.object({
```

Comment:
I noticed we're not adding `dependency` or `resourceType` here even though they're on the ddb object: should we be?

Reply:
this is the object we return as response for a get, we dont want to send those fields as part of the response

Reply:
We should be removing those fields because they're underlying implementation details of our DB. We can't pass those fields to the API users and remove those fields later on because that will be a breaking change. It's best to not pass those fields to the response now. Also we've already started removing those fields from the response of other resource entities. It'll be inconsistent if we make an exception in this case.